src/utils/toSøknadJson.ts

```diff
@@ -17,52 +26,84 @@ function formatPeriod(period: Periode): Periode {
     };
 }
 
-function periodeMedKvp({ deltarIKvp, periodeMedKvp }: Søknad) {
-    if (deltarIKvp) {
-        return formatPeriod(periodeMedKvp as Periode);
+function kvalifiseringsprogram({ deltar, periode }: Kvalifiseringsprogram) {
+    if (deltar) {
+        return { deltar, periode: formatPeriod(periode as Periode) };
     }
-    return null;
+    return { deltar };
 }
 
-function periodeMedIntro({ deltarIIntroprogrammet, periodeMedIntroprogrammet }: Søknad) {
-    if (deltarIIntroprogrammet) {
-        return formatPeriod(periodeMedIntroprogrammet as Periode);
+function introduksjonsprogram({ deltar, periode }: Introduksjonsprogram) {
+    if (deltar) {
+        return { deltar, periode: formatPeriod(periode as Periode) };
     }
-    return null;
+    return { deltar };
 }
 
-function pensjon({ mottarEllerSøktPensjonsordning, pensjon }: Søknad) {
+function pensjon(pensjonsornding: Pensjonsordning) {
+    const { mottarEllerSøktPensjonsordning, periode, utbetaler } = pensjonsornding;
     if (mottarEllerSøktPensjonsordning) {
-        return { ...pensjon, periode: formatPeriod(pensjon.periode) };
+        return { utbetaler, mottarEllerSøktPensjonsordning, periode: formatPeriod(periode) };
     }
-    return pensjon;
+    return pensjonsornding;
 }
 
-function etterlønn({ mottarEllerSøktEtterlønn, etterlønn }: Søknad) {
+function etterlønn(etterlønn: Etterlønn) {
+    const { periode, mottarEllerSøktEtterlønn, utbetaler } = etterlønn;
     if (mottarEllerSøktEtterlønn) {
-        return { ...etterlønn, periode: formatPeriod(etterlønn.periode) };
+        return { mottarEllerSøktEtterlønn, utbetaler, periode: formatPeriod(periode) };
     }
     return etterlønn;
 }
```

Comment:
Samme her, kan disse samles? Eller er det en grunn til at vi har 'duplikater'? 😊

Reply:
Jeg tenker egentlig at de er fine å ha separate? Selv om de er veldig like i datastruktur er det fint å kunne endre på logikk for etterlønn, pensjonsordning og institusjonsopphold uten at de påvirker hverandre

Reply:
Allright! Enig :)